Installing anaconda Instruction

emma0925 opened the pull request:
This PR reconstructed and elaborated on the existing Setup_conda.md. 
- Moved the introduction of Conda to a new file 'Introduce_Conda.md' and added a redirect link to the Setup_conda.md
- Add the Installation Instruction for Anaconda for both MacOS and Windows, which wasn't mentioned in the originial Setup_conda.md

YankeMao reviewed (commented):
There are a few things should be improved:
1. There are some pictures waiting to be added for the section about the installation of the Anaconda on MacOS. 
2. The re-direct link in the section about the installation of the Anaconda on MacOS would re-direct to the wrong position. It should re-direct to the sections about the installation of the Anaconda on Windows and Linux respectively.
3. When describing the installation of Anaconda, it is worth giving some suggestions such as remembering to add conda to the environment path during installation,
4. the content on the install_conda.md and the Setup_Conda.md were overlapped. Only need to keep one of them.

emma0925 commented:
> There are a few things should be improved:
> 
> 1. There are some pictures waiting to be added for the section about the installation of the Anaconda on MacOS.
> 2. The re-direct link in the section about the installation of the Anaconda on MacOS would re-direct to the wrong position. It should re-direct to the sections about the installation of the Anaconda on Windows and Linux respectively.
> 3. When describing the installation of Anaconda, it is worth giving some suggestions such as remembering to add conda to the environment path during installation,
> 4. the content on the install_conda.md and the Setup_Conda.md were overlapped. Only need to keep one of them.

Thank you for your comment.
- I had fixed the images path, so it is displaying now. 
- I also fix the redirect link. 
- mentioned to add conda to the environment path during installation
-  Removed the duplicated  file intallation.md

muyuxuan946 commented:
Hey Emma, considering the extensive nature of this document, it might be more convenient to split it into several files, each addressing distinct sections or topics. Such as install and setup can be in two separate files.  Additionally, to enhance navigability, incorporating a table of contents at the beginning can provide a clear roadmap for easy reference. This approach would ensure better organization and accessibility of information.

muyuxuan946 commented:
Oh also, there is no pr description. It's better to add one to clarify.

emma0925 commented:
@muyuxuan946 @YankeMao I have revised according to your comments. Please take another look. Thanks

YankeMao commented on Topics/Development_Process/Conda/Setup_Conda.md:
```diff
@@ -60,7 +28,7 @@ Then just download and click through the install process until you have complete
 
 The setup guide for **MacOS** is [here](#setup-macos), for **Windows** is [here](#setup-windows)
```
These two hyperlinks don't work. The paths were wrong.

YankeMao commented:
Good job! Everything seems OK. Approved merge

YankeMao reviewed (approved):
Good Job! Approved
